contracts/Compressor.sol

```diff
@@ -7,19 +7,7 @@ import {ISystemContract} from "./interfaces/ISystemContract.sol";
 import {Utils} from "./libraries/Utils.sol";
 import {UnsafeBytesCalldata} from "./libraries/UnsafeBytesCalldata.sol";
 import {EfficientCall} from "./libraries/EfficientCall.sol";
-import {
-    L1_MESSENGER_CONTRACT,
-    INITIAL_WRITE_STARTING_POSITION,
-    COMPRESSED_INITIAL_WRITE_SIZE,
-    STATE_DIFF_ENTRY_SIZE,
-    STATE_DIFF_ENUM_INDEX_OFFSET,
-    STATE_DIFF_FINAL_VALUE_OFFSET,
-    STATE_DIFF_DERIVED_KEY_OFFSET,
-    DERIVED_KEY_LENGTH,
-    VALUE_LENGTH,
-    ENUM_INDEX_LENGTH,
-    KNOWN_CODE_STORAGE_CONTRACT
-} from "./Constants.sol";
+import {L1_MESSENGER_CONTRACT, INITIAL_WRITE_STARTING_POSITION, COMPRESSED_INITIAL_WRITE_SIZE, STATE_DIFF_ENTRY_SIZE, STATE_DIFF_ENUM_INDEX_OFFSET, STATE_DIFF_FINAL_VALUE_OFFSET, STATE_DIFF_DERIVED_KEY_OFFSET, DERIVED_KEY_LENGTH, VALUE_LENGTH, ENUM_INDEX_LENGTH, KNOWN_CODE_STORAGE_CONTRACT} from "./Constants.sol";
```

Comment:
If possible, I would personally turn off the formatting part that puts multi-line imports into one line. This line will definitely not fit into a screen and will be harder to amend

Reply:
https://prettier.io/docs/en/rationale.html#imports